src/test/java/redis/clients/jedis/commands/jedis/JedisCommandsTestBase.java

```diff
@@ -1,29 +1,49 @@
 package redis.clients.jedis.commands.jedis;
 
+import java.util.Arrays;
+import java.util.Collection;
+
 import org.junit.After;
 import org.junit.Before;
+import org.junit.runners.Parameterized.Parameters;
 
 import redis.clients.jedis.DefaultJedisClientConfig;
 import redis.clients.jedis.HostAndPort;
 import redis.clients.jedis.Jedis;
 import redis.clients.jedis.HostAndPorts;
-import redis.clients.jedis.util.RedisProtocolUtil;
+import redis.clients.jedis.RedisProtocol;
 
 public abstract class JedisCommandsTestBase {
 
+  /**
+   * Input data for parameterized tests. In principle all subclasses of this
+   * class should be parameterized tests, to run with the default protocol,
+   * with RESP2 and with RESP3.
+   */
+  @Parameters
+  public static Collection<Object[]> data() {
+    return Arrays.asList(
+        new Object[]{ null },
```

Comment:
Null means that it's gonna use the default RESP version chosen by the server (which is 2 as far as I know). Arguably it does not make sense to run like that, since we also explicitly run with v2. But I don't want to change in this PR _what_ tests are running, just _how_ they run.

Reply:
Could you put a note next to the parameter, to explain that we also want to test defaults?
I was not able to quickly track this down, so it might help us in the future to decipher the code.